prac_05/hex_colours.py

```diff
@@ -1,19 +1,13 @@
 
 COLOUR_TO_CODE= {"Amethyst": "#9966cc", "Baby Pink": "#f4c2c2","Canary Yellow": "#ffef00", "Cosmic Latte": "#fff8e7",
                  "Earth Yellow": "#e1a95f", "Granny Smith Apple": "#a8e4a0", "Lavender Blue": "#ccccff", "Moss Green": "#8a9a5b",
-                 "SaddleBrown": "#8b4513","Zomp": "#39a78e" }
+                 "SaddleBrown": "#8b4513","Zomp": "#39a78e", "Yellow Orange": "#ffae42" }
```

Comment:
Are the "SaddleBrown" and "Zomp" colours intentionally doubled and "Yellow Orange" added?